crates/types/src/joints/body.rs

```diff
@@ -125,3 +136,45 @@ impl Div<f32> for BodyJoints<f32> {
         }
     }
 }
+
+#[derive(
+    Clone, Copy, Debug, Default, Deserialize, PartialEq, Eq, Serialize, SerializeHierarchy,
+)]
+#[serialize_hierarchy(bound = "T: SerializeHierarchy + Serialize, for<'de> T: Deserialize<'de>")]
+pub struct LowerBodyJoints<T> {
+    pub left_leg: LegJoints<T>,
+    pub right_leg: LegJoints<T>,
+}
```

Comment:
Naming suggestion: `LegPairJoints` and `ArmPairJoints`

Reply:
I also thought about that while implementing the types. My thoughts on that were that it is not clear what part is actually still body or which is already leg/arm. For example the hip-yaw-pitch is not clearly belonging to any leg of the NAO robot.
I personally like the Lower and Upper Body more.

Reply:
My thought was quite simple: it containt two `LegJoints`. `LowerBodyJoints` feels a bit vague. But you decide...